docs/index.md

```diff
@@ -20,25 +20,25 @@ hero:
 
 features:
   - title: Modular Architecture
-    details: PX4 is highly modular and extensible both in terms of hardware and software. It utilizes a port-based architecture � which means when developers add components, the extended system does not lose robustness or performance. 
+    details: PX4 is highly modular and extensible both in terms of hardware and software. It utilizes a port-based architecture � which means when developers add components, the extended system does not lose robustness or performance.
   - title: Open Source
     details: PX4 is co-developed with a global development community. The flightstack is not just fulfilling the needs of one lab or one company, but has been intended as a general toolkit and is widely used and adopted in the industry.
   - title: Configurability
     details: PX4 offers optimised APIs and SDKs for developers working with integrations. All the modules are self-contained and can be easily exchanged against a different module without modifying the core. Features are easy to deploy and reconfigure.
   - title: Autonomy Stack
     details: PX4 is designed to be deeply coupled with embedded computer vision for autonomous capabilities . The framework lowers the barrier of entry for developers working on localization and obstacle detection algorithms.
   - title: Real-world Validation
-    details: Thousands of commercial PX4 based systems have been deployed worldwide. In parallel, dedicated flight test team clocking up thousands of flight hours each month running hardware and software tests to ensure the codebase�s safety and reliability.
+    details: Thousands of commercial PX4 based systems have been deployed worldwide. In parallel, dedicated flight test team clocking up thousands of flight hours each month running hardware and software tests to ensure the codebase�s safety and reliability.
```

Comment:
What's the odd change here in codebases?

Reply:
The file `docs/index.md` was apparently safed in some UTF-8 incompatible code which wasn't recognized by vscode. Now I just manually changed the characters and saved as UTF-8 in the first commit. I didn't see other files being affected by this at least not in the diff of this pull request.